tests/exampleRepoTests/python3-v2.smoke.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs-extra';
+
+describe('nodeJS-v2', () => {
+  let swaggerJson: any = {};
+
+  beforeAll(async () => {
+    const fileText = await fs.readFile('./test-repos/python3-v2/swagger/swagger.py', 'utf8');
+    swaggerJson = JSON.parse(fileText.slice(60));
+    // change the 60 char if the comment at the start of the swagger.py file changes
```

Comment:
Could you explain this one to me? There's probably a simpler way to do this, i.e. slicing the file at the index where we find whatever the python comment is

Reply:
That way it would always be dynamic